pkg/common/telemetry/names.go

```diff
@@ -184,7 +312,7 @@ const (
 	JoinToken = "join_token"
 
 	// JWTKey functionality related to a JWT key; should be used with other tags
-	// to add clarity
+	// to add clarity. Should NEVER actually provide the key itself, use Key ID instead.
```

Comment:
Hmm... Should we take this chance to change calls to this to use Kid instead? This updated comment has got me worried about future misuse, and I don't see any value in keeping `jwt_key` when we have `kid` already and the value of the latter is really what we want anyway

Reply:
so the only 2 places this is used are in ca_manager.go, declaring "jwt_key" "prepare" and "jwt_key" "activate"; so it is already used safely and only to declare "some activity involving a jwt key"

Reply:
Yes understand, it is the "future misuse" part I was concerned about... the "don't leave knives laying around" rule. In the end though I won't block on it since it's the current behavior.

Thanks for fixing the imports!